omero/conf.py

```diff
@@ -82,9 +82,13 @@
     'commit' : (omero_github_root + 'commit/%s', ''),
     'omedocs' : (doc_github_root + '%s', ''),
     # API links
-    'javadoc' : (downloads_root + '/latest/omero5.1/api/%s', ''),
-    'pythondoc' : (downloads_root + '/latest/omero5.1/api/python/%s', ''),
-    # Virtual Appliance downloads
+    'javadoc' : (downloads_root + '/latest/omero/api/%s', ''),
+    'pythondoc' : (downloads_root + '/latest/omero/api/python/%s', ''),
+    # Doc links
+    'bf_doc' : (oo_site_root + '/support/bio-formats/%s', ''),
+    # Downloads
+    'downloads' : (downloads_root + '/latest/omero/%s', ''),
+    'bf_downloads' : (downloads_root + '/latest/bio-formats/%s', ''),
```

Comment:
Ah, you're right, I grepped to check if it was used in the other two docs sets and forgot to check for OMERO